docs/en/models/yolov6.md

```diff
@@ -20,15 +20,34 @@ keywords: Meituan YOLOv6, object detection, real-time applications, BiC module,
 - **Enhanced Backbone and Neck Design:** By deepening YOLOv6 to include another stage in the backbone and neck, this model achieves state-of-the-art performance on the COCO dataset at high-resolution input.
 - **Self-Distillation Strategy:** A new self-distillation strategy is implemented to boost the performance of smaller models of YOLOv6, enhancing the auxiliary regression branch during training and removing it at inference to avoid a marked speed decline.
 
+## Supported Tasks and Modes
+
+The YOLOv6 series offers a range of models, each optimized for high-performance [Object Detection](../tasks/detect.md). These models cater to varying computational needs and accuracy requirements, making them versatile for a wide array of applications.
+
+| Model Type | Tasks Supported                        | Inference | Validation | Training | Export |
+| ---------- | -------------------------------------- | --------- | ---------- | -------- | ------ |
+| YOLOv6     | [Object Detection](../tasks/detect.md) | ✅        | ✅         | ✅       | ✅     |
+
+This table provides a detailed overview of the YOLOv6 model variants, highlighting their capabilities in object detection tasks and their compatibility with various operational modes such as [Inference](../modes/predict.md), [Validation](../modes/val.md), [Training](../modes/train.md), and [Export](../modes/export.md). This comprehensive support ensures that users can fully leverage the capabilities of YOLOv6 models in a broad range of object detection scenarios.
+
+!!! warning "Configuration Files"
+
+    YOLOv6 models are supported in Ultralytics only through `.yaml` configuration files. These files contain the model architecture, training parameters, and other essential details required to build and train the model.
+
 ## Performance Metrics
 
-YOLOv6 provides various pre-trained models with different scales:
+!!! Performance
 
-- YOLOv6-N: 37.5% AP on COCO val2017 at 1187 FPS with NVIDIA Tesla T4 GPU.
-- YOLOv6-S: 45.0% AP at 484 FPS.
-- YOLOv6-M: 50.0% AP at 226 FPS.
-- YOLOv6-L: 52.8% AP at 116 FPS.
-- YOLOv6-L6: State-of-the-art accuracy in real-time.
+    === "Detection"
```

Comment:
same as above

Reply:
We just have `.yaml` for YOLOv7 we don't need coco here!

Reply:
But in the `.yaml` we have the number of classes of the coco, so maybe let's add this!

Reply:
but we have the coco metrics in the table right🤔

Reply:
Yes, the best solution here is to add the (COCO) postfix!